packages/create-next-app/helpers/examples.ts

```diff
@@ -17,8 +11,15 @@ export type RepoInfo = {
 }
 
 export async function isUrlOk(url: string): Promise<boolean> {
-  const res = await got.head(url).catch((e) => e)
-  return res.statusCode === 200
+  try {
+    const res = await fetch(url, { method: 'HEAD' })
+    if (res.status !== 200) {
+      return false
+    }
+    return true
```

Comment:
```suggestion
    return res.status === 200
```

Reply:
Addressed in [`786162b` (#58030)](https://github.com/vercel/next.js/pull/58030/commits/786162b26394a37fcf1a2d63c3cedc99a5efb911).